Sources/Target/Linux/Thread.cpp

```diff
@@ -105,8 +105,8 @@ ErrorCode Thread::updateStopInfo(int waitStatus) {
     ProcessThreadId ptid(process()->pid(), tid());
     ErrorCode error = process()->ptrace().getSigInfo(ptid, si);
     if (error != kSuccess) {
-      DS2LOG(Warning, "unable to get siginfo_t for tid %d, error=%s", tid(),
-             strerror(errno));
+      DS2LOG(Warning, "unable to get siginfo_t for tid %d, errno=%s", tid(),
+             Stringify::Errno(errno));
```

Comment:
Not really. For instance, `ENOENT` would be translated to "No such file or directory" even if we're not using files. Having the errno makes it easier to grep for it in man pages, etc. Also, the formatting with `strerror` looks stupid.